code/modules/mob/mob.dm

```diff
@@ -696,7 +696,7 @@
 			stat(null, "Next Map: [cached.map_name]")
 		stat(null, "Round ID: [GLOB.round_id ? GLOB.round_id : "NULL"]")
 		stat(null, "Server Time: [time2text(world.timeofday, "YYYY-MM-DD hh:mm:ss")]")
-		stat(null, "Round Time: [worldtime2text()]")
+		stat(null, "Round Time: [gameTimestamp("hh:mm:ss", (world.time - SSticker.round_start_time))]")
```

Comment:
```suggestion
		stat(null, "Round Time: [gameTimestamp("hh:mm:ss", (world.time - (SSticker.round_start_time || world.time)))]")
```

This is a short hand hacky way to make it evaluate to 0 if SSticker.round_start_time is 0 so that it shows 00:00:00 during the lobby. (the `||` operator will look at `SSticker.round_start_time` and if it's 0 or null, use world.time instead, so it evaluates to `world.time-world.time`)

Another way would be to do this:

```suggestion
		if (SSticker.round_start_time)
				stat(null, "Round Time: [gameTimestamp("hh:mm:ss", (world.time - SSticker.round_start_time))]")
		else
				stat(null, "Round Time: [gameTimestamp("hh:mm:ss", 0)]")
```

or even 

```suggestion
		if (SSticker.round_start_time)
				stat(null, "Round Time: [gameTimestamp("hh:mm:ss", (SSticker.round_start_time ? world.time - SSticker.round_start_time : 0))]")

```

Reply:
Another solution would be to make it check the current ticker state, and if its in lobby, just rename it from Round Time, to Lobby Time

Reply:
Works for me, I just wanted to be as minimalist as possible because, while i'm not sure what the actual performance hit of this is, every single player on a server has one and it's updated every tick.

Reply:
Stats are updated every 8 ticks.

Reply:
Oh, i'm dumb. Good to know.